src/clj/game/cards/ice.clj

```diff
@@ -1242,7 +1242,7 @@
 
    "Kakugo"
    {:events {:pass-ice {:async true
-                        :req (req (= target card))
+                        :req (req (= (:cid target) (:cid card)))
```

Comment:
We have a `same-card?` helper function for this as well :)

Reply:
updated.

Reply:
thanks!